### ISSUE TITLE
Frage: Organization Profile löschen

### ISSUE THREAD

**Issue by StefanieD** [cross-solution/YAWIK#585]
Hallo,
ist es möglich als Admin einer Firma ein Firmenprofil zu löschen?
Ich habe diese Funktion nirgends gefunden. Ist es in Planung?

**Comment by cbleek** [comment]
Was meinst du mit "Firmenprofil öschen?"

Ein Firmenprofil ist ja nur eine Ansicht auf die Daten einer Firma. Es gibt die Möglichkeit die Sicht auf die Firmendaten zu aktivieren wenn:

- wenn die Firma Jobs hat
- immer
- nie

**Comment by StefanieD** [comment]
Ja, Firmenprofil ist hier das falsche Wort. Ich meine eine Firma allgemein. 
Kann ich als Admin oder als "Besitzer" eine Firma löschen? 

Ich hatte dazu ein Issue gefunde, wo vorgeschlagen wurde, die Löschfunktion mit in die Tabelle (in die letzte Spalte) zu integrieren. Aber ich finde es gerade nicht mehr.

**Comment by cbleek** [comment]
@TiSiE kannst du Stefani bitte erklären, wie das bei den Jobs funktioniert und was gemacht werden muss.

**Comment by StefanieD** [comment]
Bitte nicht missverstehen. Es geht mir nicht um die Löschung von Jobs, sondern von Firmen/Organisationen.

**Comment by TiSiE** [comment]
Das Problem ist, dass an einer Firma viele Dinge dranhängen
* Jobs
  * Bewerbungen
* Employees (User)
* Settings
* Permissions

---
Bei den Jobs werden zumindest auf der Kommandozeile  auch die Abhängigkeiten gelöscht.  
In der WebUI aber nicht. Dort aber bei den Usern.

Was also getan werden muss:

1. Abhängigkeiten einer Firma ermitteln
2. Analog zu Usern im WebUI und Jobs auf der Kommandozeile das Löschen implementieren.

**Comment by StefanieD** [comment]
Ok, verstehe ich. 
Ist von euch geplant, dass irgendwann mit einzubauen oder überlasst ihr das dann dem jeweiligen Projekt?
